update_dockerfiles

```diff
@@ -21,19 +21,23 @@ function update_extra_repositories_and_packages {
     case ${os}+${release} in
```

Comment:
Maybe add such a comment somewhere over here:
`almalinux has "curl" already so we need to install it only for other distros`